CMakeLists.txt

```diff
@@ -328,7 +343,7 @@ else()
 
     if(QUIC_ENABLE_SANITIZERS)
         message(STATUS "Configuring sanitizers")
-        list(APPEND QUIC_COMMON_FLAGS -fsanitize=address,leak,undefined -fsanitize-address-use-after-scope -Og -g3 -fno-omit-frame-pointer -fno-optimize-sibling-calls)
+        list(APPEND QUIC_COMMON_FLAGS -fsanitize=address,leak,undefined -fsanitize-address-use-after-scope -Og -ggdb -fno-omit-frame-pointer -fno-optimize-sibling-calls)
```

Comment:
I think we would prefer -ggdb3 so we would still have macros. Same with the -ggdb above.

Reply:
ACK

Reply:
-Og is most likely not what we want also (I made this mistake before, and eventually we reverted). -Og is equivalent to -O1 and some -g, so some variables are still optimized out. -O0 -g3 -ggdb is much better :)